### ISSUE TITLE
Missing information

### ISSUE THREAD

**Issue by P-Stenbumling** [veista/smartthings#36]
Im missing a lot of information from entities
Freezer:  RZ32M713E7F
Refrigerator: RR39M73657F
![image](https://user-images.githubusercontent.com/39261692/188823063-89d8fbee-f744-4a1d-a9b6-a292783807bb.png)

**Comment by veista** [comment]
Yes. The current integration is doing some bubblegum fixing for these issues. A better library is in the works. I'm just having a lack of time and since I do not have these devices the progress is not fast, I am sorry for this.

**Comment by dannytrigo** [comment]
Is the new library in another repository? Happy to help out if you need it

**Comment by veista** [comment]
I have not decided yet. I will propably add it to this since you can not get rid of the smarthings ide etc. Will propably need a lot of help when Istart adding entities to the new integration.
